tests/text/test_tabstops.py

```diff
@@ -87,6 +92,27 @@ def leader_get_fixture(self, request):
         expected_value = getattr(WD_TAB_LEADER, member)
         return tab_stop, expected_value
 
+    @pytest.fixture(params=[
+        ('w:tab',                            'SPACES',
+         'w:tab'),
+        ('w:tab',                            'DOTS',
+         'w:tab{w:leader=dot}'),
+        ('w:tab{w:pos=720}',                 'SPACES',
+         'w:tab{w:pos=720}'),
+        ('w:tab{w:pos=720}',                 'DOTS',
+         'w:tab{w:pos=720,w:leader=dot}'),
+        ('w:tab{w:pos=720,w:leader=dots}',   'SPACES',
+         'w:tab{w:pos=720}'),
+        ('w:tab{w:pos=720,w:leader=hyphen}', 'DOTS',
+         'w:tab{w:pos=720,w:leader=dot}')
+    ])
```

Comment:
Check out how I modified these cases. Also I added one for a `None` case since that's a legitimate assignment value for an optional attribute.